website/static/js/pages/project-settings-page.js

```diff
@@ -60,6 +60,51 @@ $(document).ready(function() {
         ProjectSettings.getConfirmationCode(ctx.node.nodeType);
     });
 
+    $('#discussionsSub').on('click', function() {
+        ProjectSettings.subscribeDiscussions(ctx.node.nodeType);
+    });
+
+    $('#discussionsUnsub').on('click', function() {
+        ProjectSettings.unsubscribeDiscussions(ctx.node.nodeType);
+    });
+
+    $('#enableDiscussions').on('click', function() {
+        ProjectSettings.enableDiscussions(ctx.node.nodeType);
+    });
+
+    $('#disableDiscussions').on('click', function() {
+        ProjectSettings.disableDiscussions(ctx.node.nodeType);
+    });
+
+    $('#discussionsSettings').on('submit', function() {
+        var $discussionsMsg = $('#configureDiscussionsMessage');
+
+        var $this = $(this);
+        var discussionsSub = $this.find('input[name="discussionsSub"]:checked').val();
+
+        $osf.postJSON(
+            ctx.node.urls.api + 'discussions/sub/',
+            {discussionsSub: discussionsSub}
+        ).done(function () {
+                $discussionsMsg.addClass('text-success');
+                $discussionsMsg.text('Successfully updated subscription.');
+                window.location.reload();
```

Comment:
It seems odd to reload the page after an AJAX call - would it be possible to refresh the viewmodel instead?

Perhaps moving `categorySettingsVM` out one level of scope would allow you to call an update method on it?